actors/builtin/miner/miner_actor.go

```diff
@@ -59,6 +59,7 @@ func (a Actor) Exports() []interface{} {
 		15:                        a.ReportConsensusFault,
 		16:                        a.WithdrawBalance,
 		17:                        a.ConfirmSectorProofsValid,
+		18:                        a.ChangeMultiaddrs,
```

Comment:
How would that work in practice? Either they're always multiaddrs or we'd need something to indicate what they are.

Reply:
Probably hackily. I'm not saying it would be good, just that the actors don't care. If for some unlikely reason something like that was wanted, no code here would change but the names would be misleading. 

Taken to the extreme, the field might as well be called "arbitrary provider metadata" and embed JSON. But "addresses" is fine and this is not a blocking issue for me.

Reply:
I've prefixed with `Multi` to avoid confusion with methods that use `address.Address`s -- `ControlAddresses` & `ChangeWorkerAddress`. Having a method called `ChangeAddresses` seems confusing given that it operates on something completely different that `ControlAddresses`.

Reply:
At that point, I'd add additional fields. Although, multiaddrs are extensible enough (literally multicodec + argument) that I'd expect us to always use mutliaddrs here (even if it means assigning new multicodecs for new multiaddrs).